templates/test/ci/prow-azurediskcsi-migration-off/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+resources:
+  - ../prow-user-assigned-managed-identity
```

Comment:
why are these starting from a user-assigned template?

Reply:
I figured out that we did not had a top level user assigned identity template. So added this to avoid repetition and use in other templates that needs a user assigned identity.  

I took inspiration from external cloud provider test that Jack wrote sometime back which is using user assigned identity. 

If we do not add this template, then in every template we will need to add following to the patch 
```
patchesStrategicMerge:
  - ../patches/uami-md-0.yaml
  - ../patches/uami-control-plane.yaml
 
 ```